src/glyph/serialize.rs

```diff
@@ -236,12 +238,8 @@ impl Image {
     fn to_event(&self) -> Event {
         let mut start = BytesStart::borrowed_name(b"image");
         start.push_attribute(("fileName", self.file_name.to_str().unwrap_or("missing path")));
-        start.push_attribute(("xScale", self.transform.x_scale.to_string().as_str()));
-        start.push_attribute(("yScale", self.transform.y_scale.to_string().as_str()));
-        start.push_attribute(("xyScale", self.transform.xy_scale.to_string().as_str()));
-        start.push_attribute(("yxScale", self.transform.yx_scale.to_string().as_str()));
-        start.push_attribute(("xOffset", self.transform.x_offset.to_string().as_str()));
-        start.push_attribute(("yOffset", self.transform.y_offset.to_string().as_str()));
+
+        transformation_to_attributes(&mut start, &self.transform);
```

Comment:
I would call this function something like `write_transform_attributes`; the `to` makes it sound like this is returning something?